includes/polling/applications/bird2.inc.php

```diff
@@ -5,8 +5,9 @@
 use LibreNMS\Util\Oid;
 
 $name = 'bird2';
+$mib = 'NET-SNMP-EXTEND-MIB';
 
-$birdOutput = snmp_get($device, 'nsExtendOutputFull.' . Oid::encodeString($name), '-Oqv', 'NET-SNMP-EXTEND-MIB');
+$birdOutput = SnmpQuery::get($mib . '::nsExtendOutputFull.' . Oid::encodeString($name))->value();
```

Comment:
```suggestion
$birdOutput = SnmpQuery::get('NET-SNMP-EXTEND-MIB::nsExtendOutputFull.' . Oid::encodeString($name))->value();
```

This seems like the format we should use for all of them.  It includes the mib so we get the correct display hints and output formatting for SnmpQuery.  It also removes the magic number style numeric oids.

Reply:
Does not fix it.

```
if (! empty($agent_data['app'][$name])) {
    $rawdata = $agent_data['app'][$name];
} else {
    $options = '-Oqv';
    $oid = '.1.3.6.1.4.1.8072.1.3.2.3.1.2.10.102.114.101.101.114.97.100.105.117.115';
//    $rawdata = snmp_get($device, $oid, $options);
    $rawdata = SnmpQuery::get('NET-SNMP-EXTEND-MIB::nsExtendOutputFull.' . Oid::encodeString($name))->value();
}
```

Gives me the exact same error with it only returning the first line.

Reply:
Alternatively, we just fetch `NET-SNMP-EXTEND-MIB::nsExtendOutputFull` and cache it. Then use that for all apps.

Reply:
Sure